stock_move_common_dest/tests/test_move_common_dest.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests import SavepointCase
+
+
+class TestCommonMoveDest(SavepointCase):
```

Comment:
Could you switch to TransactionCase ?

Reply:
Changed. Sorry, I migrated this module because is used by https://github.com/OCA/wms/pull/446 and this last one have multiple dependencies. This implies that I am reviewing the result of the tests little by little